Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -142,18 +142,18 @@ base
 ----
 
 The base is the base domain to run a site on. It either accepts a fully qualified URL or a relative segment "/" to react to any domain name.
-It is possible to set a site base prefix to just "/site1" and "/site2" or "www.mydomain.com" instead of entering a full URI.
+It is possible to set a site base prefix to just "/site1" and "/site2" or "example.com" instead of entering a full URI.
```

Comment:
```suggestion
It is possible to set a site base prefix to just :samp:`/site1`, :samp:`/site2` or even :samp:`example.com` instead of entering a full URI.
```